docs/tables/aws_cloudwatch_metric_statistic_data_point.md

```diff
@@ -0,0 +1,190 @@
+# Table: aws_cloudwatch_metric_statistic_data_point
+
+This table provides statistic data points for the specified metric.
+
+The maximum number of data points returned from a single call is 1,440. If you request more than 1,440 data points, CloudWatch returns an error. To reduce the number of data points, you can narrow the specified time range and make multiple requests across adjacent time ranges, or you can increase the specified period. Data points are not returned in chronological order.
+
+If you need to fetch more than 1440 data points then please use the `aws_cloudwatch_metric_data_point` table.
+
+- You **_must_** specify `metric_name`, `namespace`, `timestamp` and `dimensions` in a `where` clause in order to use this table. To fetch aggregate statistics data `dimensions` is not required.
+
+- You need to provide the timestamp value in a range like `timestamp >= '2023-03-15T00:00:00Z' and timestamp <= '2023-03-15T13:33:00Z' or like in the examples below.`
```

Comment:
Can we update the table to default a start time to 24 hours ago and an end time to current time if we can't work it out based on quals? If so, we can then update the details here

Reply:
Also, if we have a reasonable way to work out a default start/end time, could we make `timestamp` qual optional? So if a user only specifies `metric_name` and `namespace`, we could get results from the past 24 hours.

What period would we work out by default in that case?

Reply:
Updated the default end time to the current time and the default start time to 24hrs ago.
Made timestamp as an optional column, so if the user does not provide it, we can get the result from the last 24 hrs.
The default period will be 60 sec, fetching exactly 1440 datapoints.